src/AzSK.ADO/Framework/Core/SVT/SVTControlAttestation.ps1

```diff
@@ -282,6 +282,21 @@ class SVTControlAttestation
 					$controlState.State.ApprovedExceptionID = $this.attestOptions.ApprovedExceptionID
 					$controlState.State.ExpiryDate = $exceptionApprovalExpiryDate.ToString("MM/dd/yyyy");
 				}
+
+				# Checking if the resource id is present in extended expiry list of control settings
+				if([Helpers]::CheckMember($this.ControlSettings, "ExtendedExpiryResources") -and [Helpers]::CheckMember($this.ControlSettings, "ExtendedExpiryDuration")){
+					if(($this.ControlSettings.ExtendedExpiryResources | Get-Member "ResourceType") -and ($this.ControlSettings.ExtendedExpiryResources | Get-Member "ResourceIds")) {
```

Comment:
Has this been tested in case ExtendedExpiryResources  is empty in control settings?

Reply:
yes, working as expected